### PR TITLE
typo: 9:00AM => 9:00 AM

### PR DESCRIPTION
**Pull request by detailyang**
Hello.

See this [openresty.org/pull/53](https://github.com/openresty/openresty.org/pull/53), we should use `9:00 AM` rather than `9:00AM`

Also according the [中文文案排版指北](https://github.com/sparanoid/chinese-copywriting-guidelines), may be we should add the blank between Chinese and Number?

Guys What do you think of it :)

### CODE REVIEW

**Comment by agentzh** [comment]
+1

File: cn/2016/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
 	<!-- <meta name="renderer" content="webkit|ie-comp|ie-stand"> -->
 	<meta name="description" content="OpenResty基金会发起，腾讯大讲堂和珠海开源软件技术促进中心联合主办的OpenrestyCon 2016
```
Comment (moonbingbing) [inline cn/2016/index.html]: also need spaces around `OpenResty`

**Comment by moonbingbing** [comment]
merged. thx

**Comment by detailyang** [comment]
@moonbingbing  @agentzh 

maybe we should use [chinese-copywriting-guidelines](https://github.com/sparanoid/chinese-copywriting-guidelines) as our copywriting standard.

i'm confused about `2016年12月10日`.  According `Place one space before / after numbers`, it should add the one space. Also i have found that the Apple is following this rule in the [Apple (中国)](http://www.apple.com/cn/)
screen shot 2016-11-30 at 11 21 46

what do you think of it?

**Comment by agentzh** [comment]
@detailyang Agreed.

**Comment by moonbingbing** [comment]
I think number/date should not add space, for example : http://www.gov.cn/

2016-11-30 11:26 GMT+08:00 detailyang <notifications@github.com>:

**Comment by agentzh** [comment]
@moonbingbing Keeping a space between Chinese characters and digits/English words usually read better IMHO.

**Comment by detailyang** [comment]
No matter what style we choose， just keep the same style. Maybe it's good enough to add a repo about Chinese copywriting in [openresty](https://github.com/openresty) in order to better collaboration.

**Comment by moonbingbing** [comment]
@agentzh Agreed :)

**Pull request merged**